exchanges/bybit/bybit_websocket.go

```diff
@@ -755,7 +636,7 @@ func channelName(s *subscription.Subscription) string {
 // isSymbolChannel returns whether the channel accepts a symbol parameter
 func isSymbolChannel(name string) bool {
 	switch name {
-	case chanPositions, chanExecution, chanOrder, chanDCP, chanWallet:
+	case chanPositions, chanExecution, chanOrder, chanWallet:
 		return false
 	}
 	return true
```

Comment:
Update `subTplText` to not process auth endpoints:
```
{{ with $name := channelName $.S }}
	{{- range $asset, $pairs := $.AssetPairs }}
		{{- if isSymbolChannel $name }}
			{{- range $p := $pairs }}
				{{- $name -}} .
				{{- if eq $name "orderbook" -}} {{- $.S.Levels -}} . {{- end }}
				{{- if eq $name "kline" -}} {{- intervalToString $.S.Interval -}} . {{- end }}
				{{- $p }}
				{{- $.PairSeparator }}
			{{- end }}
		{{- end }}
	{{- end }}
	{{- $.AssetSeparator }}
{{- end }}
```

Reply:
https://github.com/thrasher-corp/gocryptotrader/pull/1670/commits/80a5edb11e8b8d4093df7c3d133def5dc2b45b60